proto-lens/src/Data/ProtoLens/Field.hs

```diff
@@ -4,8 +4,12 @@ same name.
 
 To use instances from this class, either:
 
-* Enable the @OverloadedLabels@ extension and @import Data.ProtoLens.Labels ()@,
+* Enable the @OverloadedLabels@ extension and
+  @import Data.ProtoLens.Labels ()@; or,
 * Use the 'field' function along with the @TypeApplications@ extension.
+* Import the corresponding names from the autogenerated @*_Fields@ module.
```

Comment:
Looks like we have 3 alternatives now, could you check the typography works out for this list?

Reply:
Done.  The formatting looks reasonable in haddock.  I changed it slightly to make the "or" happen in between the second and third options.